src/CarbonAware.CLI/src/Model/EmissionsDataDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CarbonAware.Model;
+
+namespace CarbonAware.CLI.Model
+{
+    public class EmissionsDataDTO
```

Comment:
I notice that this EmissionsDataDTO doesn't match the Web APIs `EmissionDataDTO`. Was that intentional?

Follow-up Q for my knowledge: if we do want them to be similar, is this a case for bringing the DTO object one level up into the CarbonAware project or was the choice intentional to have each DTO in each UI? Is that the general standard practice?

Reply:
Yes, this was intentional. Typically, DTOs are 'mapped' from domain objects but should contain data needed by the requesting client. It is generally a better practice to not tightly couple the DTOs for different clients.

Reply:
Great, thanks for explaining